diff --git a/node_modules/react-trello/dist/controllers/Lane.js b/node_modules/react-trello/dist/controllers/Lane.js
index 51cf13f..5be02c0 100644
--- a/node_modules/react-trello/dist/controllers/Lane.js
+++ b/node_modules/react-trello/dist/controllers/Lane.js
@@ -41,6 +41,8 @@ var _Draggable = _interopRequireDefault(require("../dnd/Draggable"));
 
 var laneActions = _interopRequireWildcard(require("../actions/LaneActions"));
 
+const { tunnel } = require('@appactions/driver');
+
 class Lane extends _react.Component {
   constructor(...args) {
     super(...args);
@@ -104,7 +106,8 @@ class Lane extends _react.Component {
       onCardClick && onCardClick(card.id, card.metadata, card.laneId);
       e.stopPropagation();
     });
-    (0, _defineProperty2.default)(this, "showEditableCard", () => {
+    (0, _defineProperty2.default)(this, "showEditableCard", (event) => {
+      tunnel(event).action('Button', 'click');
       this.setState({
         addCardMode: true
       });
