diff --git a/node_modules/react-trello/dist/controllers/Lane.js b/node_modules/react-trello/dist/controllers/Lane.js
index 51cf13f..7cedb07 100644
--- a/node_modules/react-trello/dist/controllers/Lane.js
+++ b/node_modules/react-trello/dist/controllers/Lane.js
@@ -41,6 +41,8 @@ var _Draggable = _interopRequireDefault(require("../dnd/Draggable"));
 
 var laneActions = _interopRequireWildcard(require("../actions/LaneActions"));
 
+const { tunnel } = require('@appactions/driver');
+
 class Lane extends _react.Component {
   constructor(...args) {
     super(...args);
@@ -104,7 +106,8 @@ class Lane extends _react.Component {
       onCardClick && onCardClick(card.id, card.metadata, card.laneId);
       e.stopPropagation();
     });
-    (0, _defineProperty2.default)(this, "showEditableCard", () => {
+    (0, _defineProperty2.default)(this, "showEditableCard", (event) => {
+      // tunnel(event).action('Button', 'click');
       this.setState({
         addCardMode: true
       });
diff --git a/node_modules/react-trello/dist/widgets/EditableLabel.js b/node_modules/react-trello/dist/widgets/EditableLabel.js
index aadc233..96cef08 100644
--- a/node_modules/react-trello/dist/widgets/EditableLabel.js
+++ b/node_modules/react-trello/dist/widgets/EditableLabel.js
@@ -13,6 +13,8 @@ var _react = _interopRequireDefault(require("react"));
 
 var _propTypes = _interopRequireDefault(require("prop-types"));
 
+const { tunnel } = require('@appactions/driver');
+
 class EditableLabel extends _react.default.Component {
   constructor({
     value: _value
@@ -23,6 +25,8 @@ class EditableLabel extends _react.default.Component {
     });
     (0, _defineProperty2.default)(this, "onTextChange", ev => {
       const value = this.getText(ev.target);
+      // this is bugged
+      // tunnel(ev).action('Input', 'type', value);
       this.setState({
         value: value
       });
